ceph.spec.in

```diff
@@ -884,6 +884,8 @@ mkdir -p %{buildroot}%{_sbindir}
 install -m 0644 -D src/logrotate.conf %{buildroot}%{_sysconfdir}/logrotate.d/ceph
 chmod 0644 %{buildroot}%{_docdir}/ceph/sample.ceph.conf
 install -m 0644 -D COPYING %{buildroot}%{_docdir}/ceph/COPYING
+#modify aio-max-nr =1048576
+install -m 0644 -D src/90-ceph-osd-sysctl.conf %{buildroot}%{_prefix}/lib/sysctl.d/90-ceph-osd-sysctl.conf
```

Comment:
use the macro of `%{_sysctldir}`. see https://fedoraproject.org/wiki/Packaging:Guidelines#binfmt.d.2C_sysctl.d_and_tmpfiles.d

Reply:
OK,I change it